README.md

```diff
@@ -13,7 +13,7 @@ With [Iceberg](https://useiceberg.com/), you get what you need to write beautifu
 
 1. Download or clone this GitHub repository into your local site's `wp-content/plugins` directory
 2. Browse to the folder using the command line
-3. Run `npm setup` to install dependencies
+3. Run `npm install` to install dependencies
```

Comment:
Let's change this to `npm run setup` instead - as that's the proper command to install npm and composer dependencies.